player/lua/osc.lua

```diff
@@ -2777,6 +2779,16 @@ mp.observe_property("idle-active", "bool",
         request_tick()
     end
 )
+
+mp.observe_property('display-fps', 'number',
+    function(_, val)
+        -- may be nil if unavailable or 0 fps is reported
+        if not val or not user_opts.tick_delay_follow_display_fps then return end
+        user_opts.tick_delay = 1 / val
+        request_tick()
+    end
+)
```

Comment:
```suggestion
mp.observe_property('display-fps', 'number', function(_, val)
    -- may be nil if unavailable or 0 fps is reported
    if not val or not user_opts.tick_delay_follow_display_fps then
        return
    end
    user_opts.tick_delay = 1 / val
end)
```

General style nits to be in line with the rest of the file. Also, is `request_tick` required? I would assume no.